scripts/zones/Spire_of_Mea/mobs/Envier.lua

```diff
@@ -7,99 +7,37 @@ mixins = { require('scripts/mixins/families/empty_terroanima') }
 ---@type TMobEntity
 local entity = {}
 
--- TODO: Check for traits like double attack and store TP.
--- TODO: Check for low HP behavior changes.
--- TODO: Immunities, EEM, Attack Speed
-
-local vars =
-{
-    SEETHER_TIME     = 'seetherTime',
-    SEETHER_ELIGIBLE = 'seetherEligible',
-}
-
-local seetherParams =
-{
-    SPAWN_PERCENT = 50,
-    SPAWN_MIN     = 10,
-    SPAWN_MAX     = 45,
-}
-
------------------------------------
--- Set the spawn time for the next Seether.
------------------------------------
-local setSeetherSpawnTime = function(mob)
-    mob:setLocalVar(vars.SEETHER_TIME, GetSystemTime() + math.random(seetherParams.SPAWN_MIN, seetherParams.SPAWN_MAX))
+local function seetherTimer(mob)
+    mob:setLocalVar('seetherTimer', GetSystemTime() + math.random(30, 90)) -- Envier summons Seethers every 30 to 90 seconds.
```

Comment:
A 1 line local function used just twice isnt really worth it. Just use the line in the 2 places its used.